mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -2769,6 +2769,20 @@ class LogisticRegressionSuite
       LogisticRegressionSuite.allParamSettings, checkModelData)
   }
 
+  test("read/write with BoundsOnCoefficients") {
+    def checkModelData(model: LogisticRegressionModel, model2: LogisticRegressionModel): Unit = {
+      assert(model.getLowerBoundsOnCoefficients === model2.getLowerBoundsOnCoefficients)
+      assert(model.getUpperBoundsOnCoefficients === model2.getUpperBoundsOnCoefficients)
```

Comment:
In ```checkModelData```, we should check model itself like ```coefficients``` and ```intercept```. We already have check equality for all params including ```lowerBoundsOnCoefficients``` inside of ```testEstimatorAndModelReadWrite```.

Reply:
Or we can merge this test case with existing read/write test.

Reply:
The existing read/write test has `elasticNetParam = 0.1`.
I can update the `checkModelData` function here to check model `coefficients` and `intercept` if it's OK with you.

Reply:
Sure, you can update existing read/write test or your test, as long as to check model itself rather than parameters.